src/handlebar-helpers.js

```diff
@@ -42,7 +42,25 @@ const helpers = {
       return x + y;
     }
     return -1;
-  }
+  },
+  substring: function (data, startIndex, endIndex) {
+    if ( typeof data  === 'string' && typeof startIndex === 'number' && typeof endIndex === 'number') {
+      return data.substring(startIndex, endIndex);
+    }
+    if ( typeof data  === 'string' && typeof startIndex === 'number') {
+      return data.substring(startIndex);
+    }
+    return ''
```

Comment:
in javascript, its ok to pass in undefined variables to a function. lets minimize the code a bit by utilizing this behavior.
```suggestion
    if ( typeof data  === 'string' && typeof startIndex === 'number' && (typeof endIndex === 'number' || endIndex === undefined )) {
      return data.substring(startIndex, endIndex);
    }
    return ''
```

Reply:
It makes sense, updating code as suggested ....

Reply:
also, make sure to update your tests according to the new return result when there are errors